package.json

```diff
@@ -17,7 +17,7 @@
   },
   "devDependencies": {
     "@changesets/cli": "^2.27.1",
-    "husky": "^8.0.3",
+    "husky": "^9.0.11",
```

Comment:
why you need to bump husky?

Reply:
The husky version was causing a bug that prevented the install of the `@strapi/blocks-react-renderer` lib that's used by the frontend to render the blocks elements fetched from strapi like images, the error was:
![Screenshot 2024-02-20 alle 09 48 35](https://github.com/pagopa/developer-portal/assets/146933303/66271868-ec0f-4ced-b222-55d131b3a04e)

Reply:
I'm not sure the bump is necessary: I just tried to run the command you executed in the screenshot starting from main, and it succeeded. Could be some misconfiguration you have on your machine?